CHANGELOG.md

```diff
@@ -44,8 +44,12 @@
   ### Bugfixes
 
 - **Service Release Process**
+
   - Fixed "None" selection issue in Technical Integration [#1161](https://github.com/eclipse-tractusx/portal-frontend/issues/1161)
 
+- **Service Management**
+  - Service Management | 400 Bad Request when Clicking 'Load More' Button [#1168](https://github.com/eclipse-tractusx/portal-frontend/issues/1168)
+
```

Comment:
Hi @ss-nikunj we have an continuous issue in the changelog file, the entry is at the wrong place and it's not even shown as conflict. I suggest you add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.